modules/hierarchical-firewall-policy/README.md

```diff
@@ -0,0 +1,249 @@
+# Google Cloud Hierarchical Firewall Policy
+
+This module allows creation of [hierarchical firewall policy](https://cloud.google.com/firewall/docs/firewall-policies) and [Rules](https://cloud.google.com/firewall/docs/firewall-policies-rule-details). It can also attach hierarchical firewall policy to multiple folders or organization. Module can create both [Cloud Firewall Essentials](https://cloud.google.com/firewall/docs/about-firewalls#firewall-essentials) and [Cloud Firewall Standard](https://cloud.google.com/firewall/docs/about-firewalls#firewall-standard) tier rules. Firewall Rules and Target folders/org attachment is optional. Hierarchical firewall policy limitations are available [here](https://cloud.google.com/firewall/docs/using-firewall-policies#limitations)
```

Comment:
> "Firewall Rules and Target folders/org attachment is optional"

Can you explain what this means?

Reply:
You can create an empty firewall policy without any rules or attaching it to any folder.

Reply:
Thanks. Can you reword it just like that? It's easier to understand.